### PR TITLE
Deprecate user sync

### PR DESCRIPTION
**Pull request by Fawke**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change
*enabledBidders*, *iframeEnabled* & *pixedEnabled* properties have been deprecated. The unit test cases have been updated to reflect that.

### CODE REVIEW

File: test/spec/userSync_spec.js [hunk 1]
```diff
@@ -250,6 +265,7 @@ describe('user sync', function () {
 
   it('should throw a warning and default to basic resgistration rules when filterSettings config is invalid', function () {
     // invalid config - passed invalid filter option
+    debugger; // eslint-disable-line
```
Comment (jaiminpanchal27) [inline test/spec/userSync_spec.js]: Remove this breakpoint

Comment (Fawke) [reply test/spec/userSync_spec.js]: Thanks for checking that, I've removed it now.

File: src/userSync.js [hunk 2]
```diff
@@ -31,7 +36,7 @@ export function newUserSync(userSyncDependencies) {
 
   // for now - default both to false in case filterSettings config is absent/misconfigured
   let permittedPixels = {
-    image: false,
+    image: true,
```
Comment (robertrmartinez) [inline src/userSync.js]: This was not mentioned in the PR,

Are we defaulting to enabling image pixels now?

Comment (Fawke) [reply src/userSync.js]: Hi, 

Yes, userSync feature had image pixels enabled by default, prior to this change. Reason for this change here is that, we removed certain deprecated fields, as a result, we lost that default setting. We're setting this value to true now to maintain the default setting.

**Pull request merged**